tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "paths": {
       "~/*": ["src/*"]
     },
-    "jsxImportSource": "react"
+    "jsxImportSource": "react",
+    "exactOptionalPropertyTypes": false
```

Comment:
It's very strict at the moment, so had to disable, since can't easily fix certain types.